build.gradle

```diff
@@ -627,6 +639,10 @@ project('standalone') {
         systemProperties System.getProperties()
         systemProperties 'logback.configurationFile' : new File('config/logback.xml').absolutePath
         systemProperties 'singlenode.configurationFile' : new File("config/standalone-config.properties").absolutePath
+        systemProperty "io.netty.tryReflectionSetAccessible", "true"
+        systemProperty "https.protocols", "TLSv1,TLSv1.1,TLSv1.2"
```

Comment:
Do we need this? My understanding is that the https.protocols property is only useful here if the client uses HttpsUrlConnection class or URL.openStream().

Reply:
Pravega has an HTTP REST API and actually I had to add this property as first fix.

Then I found errors on non HTTP I added the generic one. 
I can try to drop the http specific property

Reply:
Sounds good. Thanks.

Reply:
I tried to drop the https.protocol property but I see now failures, it seems necessary.

@ravisharda do you have time to double check?

Otherwise I think we should leave this patch in the current form.

Yesterday Oracle released a new patch version of java 11 with a bunch of fixes around TLS 1.3: I will give it a try tomorrow, maybe this patch is not needed anymore at all

Reply:
@ravisharda I was wrong.
I have re-run the tests againg with a clean env and tests are passing even dropping https.protocols

Reply:
Cool. Good to know.